src/mlpack/methods/reinforcement_learning/q_networks/categorical_dqn.hpp

```diff
@@ -43,9 +41,9 @@ using namespace mlpack::ann;
  * @tparam NetworkType The type of network used for simple dqn.
  */
 template<
-  typename OutputLayerType = EmptyLoss<>,
-  typename InitType = GaussianInitialization,
-  typename NetworkType = FFN<OutputLayerType, InitType>
+  typename OutputLayerType = mlpack::ann::EmptyLoss<>,
+  typename InitType = mlpack::ann::GaussianInitialization,
+  typename NetworkType = mlpack::ann::FFN<OutputLayerType, InitType>
```

Comment:
```suggestion
  typename OutputLayerType = ann::EmptyLoss<>,
  typename InitType = ann::GaussianInitialization,
  typename NetworkType = ann::FFN<OutputLayerType, InitType>
```
Not 100% sure on this... if it does work, the same idea could apply to the rest of the file too. :+1:

Reply:
Maybe, I do not remember changing these lines, I will test it locally before